error/src/lib.rs

```diff
@@ -252,10 +252,11 @@ impl CaliptraError {
 
     /// FMC Errors
     pub const FMC_RT_ALIAS_UNIMPLEMENTED: CaliptraError = CaliptraError::new_const(0x000F0001);
```

Comment:
Do we still need this error code? If not, remove it and re-use f0001.

Reply:
done